### PR TITLE
asuka-xp.com

### PR DESCRIPTION
**Pull request by SKEIDs**
#104299

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -4707,7 +4707,7 @@ goal.com##footer[data-sponsorship-slot] > .media + .category + time + i[data-rol
 toplanzi.com###socialShareWidget
 toplanzi.com##img[onclick="fbinvite()"]
 lunar.az##.sende_paylas
-sportnews.to,indir.vip###custom_html-4
+asuka-xp.com,sportnews.to,indir.vip###custom_html-4
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:4710]: It seems this rule does nothing.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:4710]: delete the space above "カテゴリー".
Is this an unnecessary rule?
@Yuki2718

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:4710]: Do you see visual difference by this rule?

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:4710]: Possibly I have to check with AG app than AG extension. On my Chromium + AG Ext. there's no difference.

Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:4710]: I delete this domain from that rule.

**Pull request merged**